CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 ### _Black_
 
+- Remove dependency on `regex` (#2663)
```

Comment:
Should we keep the "reduce usage of the `regex` dependency ..." entry. In my opinion it looks a little weird especially since it's _after_ we state we removed regex.

Reply:
Good point, done